lib/view/multi-tasks-view.js

```diff
@@ -59,12 +59,7 @@ class ListrReactiveTask {
                 subscriber.next(status);
             });
             this._eventEmitter.on('error', (error) => {
-                if (error) {
```

Comment:
only place where event emitter is triggered

```
        this.taskHandle(this.reporter, (err, result) => {
            if (err) {
                this._eventEmitter.emit('error', err);
            } else {
                this._eventEmitter.emit('complete', result);
            }
        });
```
so error will always be present on receiver side. Additionally 'else if' is never possible.

Reply:
merging if all tests are passing. since I mainly created for https://github.com/alexa/ask-cli/pull/340 since my unit tests was failing due to code coverage below 95%.